Client/lib/PONAPI/Client/UA/YAHC.pm

```diff
@@ -31,6 +31,11 @@ has scheme => (
     isa => 'Str',
 );
 
+has ssl_options => (
+    is  => 'rw',
+    isa => 'Ref',
+);
```

Comment:
shouldn't we have some default value to avoid some potential breakage?

Reply:
We could in principle put an empty hashref here.

Reply:
I'll take care of that.